docs/organisation-logos.md

```diff
@@ -6,29 +6,15 @@ The [Organisation logo component](https://components.publishing.service.gov.uk/c
 
 This documentation provides details on how to update and add new logos that are part of the Government Identity System.
 
-## Images required
+## Image requirements
 
-The four images below are required for the organisation logo component. The images should be added to `app/assets/images/govuk_publishing_components/crests`. The image format required is a transparent PNG with the whitespace trimmed.
+The image should be added to `app/assets/images/govuk_publishing_components/crests`. The image format required is a transparent PNG, with the whitespace trimmed, unless whitespace has been agreed with a designer.
 
 ### Naming convention
 
 - `$crest` this is usually in the format `"[department_abbreviation]_crest"` all lowercase with underscores, for example `hmrc_crest`
-- `_13px/_18px` refers to the font-size used
-- `_x2` used to provide an @2x image for screens that support it
-
-### Mobile
-
-By default, the logo image used on mobile screen sizes, will have a height of `20px`.
```

Comment:
Since we're removing the specific size guidance for images on different screen sizes, should we add some guidance on what size the logos should be now?

Reply:
@andysellick I thought about this, but it seems that all the crests have slightly different sizes (especially our new No 10 one), so I didn't want to say anything explicit

Reply:
Okay, that's fair.